### PR TITLE
GUI-2170 added page info into page title

### PR DESCRIPTION
**Pull request by dkavanagh**
added page info into page title

https://eucalyptus.atlassian.net/browse/GUI-2170

### CODE REVIEW

**Comment by euca-qa-gui-tester** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com/job/user-console-PR-monitor/1140/
Test PASSed.

**Comment by euca-qa-gui-tester** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com/job/user-console-PR-monitor/1149/
Test PASSed.

**Comment by euca-qa-gui-tester** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com/job/user-console-PR-monitor/1153/
Test PASSed.

File: eucaconsole/views/stacks.py [hunk 1]
```diff
@@ -213,7 +214,7 @@ def __init__(self, request):
 
     @view_config(route_name='stack_view', renderer=TEMPLATE)
     def stack_view(self):
-        if self.stack is None and self.request.matchdict.get('id') != 'new':
+        if self.stack is None and self.request.matchdict.get('name') != 'new':
```
Comment (kamalgill) [inline eucaconsole/views/stacks.py]: @dkavanagh: Changing the matchdict get to 'name' seems outside the scope of GUI-2170 and makes me wonder if we need a 4.2.1 ticket for this.  Your thoughts?

Comment (dkavanagh) [inline eucaconsole/views/stacks.py]: The route contained the name all along. This is only an error catching clause, so it isn't critical. It catches the case where an unknown stack name is found in the route.

**Comment by euca-qa-gui-tester** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com/job/user-console-PR-monitor/1154/
Test PASSed.
